pilot/pkg/xds/delta.go

```diff
@@ -310,6 +310,7 @@ func (s *DiscoveryServer) processDeltaRequest(req *discovery.DeltaDiscoveryReque
 	}
 
 	push := s.globalPushContext()
+	request.Start = time.Now()
```

Comment:
Rewriting Start here is completely changing the meaning of Start, which will impact metrics, etc. The field already indicates `This represents the time of adding to the PushQueue.` which is no longer accurate

Reply:
good catch